packages/next/src/server/dev/turbopack-utils.ts

```diff
@@ -129,7 +129,11 @@ function isNodeModulesIssue(issue: Issue): boolean {
 
   return (
     issue.severity === 'warning' &&
-    issue.filePath.match(/^(?:.*[\\/])?node_modules(?:[\\/].*)?$/) !== null
+    (issue.filePath.match(/^(?:.*[\\/])?node_modules(?:[\\/].*)?$/) !== null ||
+      // Ignore Next.js itself when running next directly in the monorepo where it is not inside
+      // node_modules anyway.
+      // TODO(mischnic) prevent matches when this is published to npm
+      issue.filePath.startsWith('[project]/packages/next/'))
```

Comment:
Does anyone have a better idea here?

There is a huge amount of logs generated about `next/compiled/@ampproject/toolbox-optimizer` when running the tests if I don't add this filter.